library/psa_crypto.c

```diff
@@ -1400,6 +1400,12 @@ psa_status_t psa_get_key_attributes(mbedtls_svc_key_id_t key,
                 mbedtls_free(rsa);
             }
             break;
+#else
+        case PSA_KEY_TYPE_RSA_KEY_PAIR:
+        case PSA_KEY_TYPE_RSA_PUBLIC_KEY:
+            attributes->domain_parameters = NULL;
+            attributes->domain_parameters_size = SIZE_MAX;
```

Comment:
Setting `domain_parameters_size` to `SIZE_MAX` is wrong: it's a promise that you can read `SIZE_MAX` bytes from `domain_parameters`. This might not be critical since this is a private field, and its only accessor function `psa_get_domain_parameters` takes care of that case. But at the very least, it needs to be documented in the definition of `struct psa_key_attributes_s` in `psa/crypto_struct.h`.

https://github.com/Mbed-TLS/mbedtls/pull/8644